public/app/features/dashboard/panel_editor/QueryOptions.tsx

```diff
@@ -130,7 +130,7 @@ export class QueryOptions extends PureComponent<Props, State> {
     const tooltip = `If your time series store has a query cache this option can override the default cache timeout. Specify a
     numeric value in seconds.`;
 
-    if (!datasource.meta.queryOptions?.maxDataPoints) {
+    if (!datasource.meta.queryOptions?.maxDataPoints || !datasource.meta.queryOptions?.cacheTimeout) {
```

Comment:
this is wrong (my mistake), remove the datasource.meta.queryOptions?.maxDataPoints